aireview/src/aireview/application/controllers/review_controller.py

```diff
@@ -16,4 +16,4 @@ def __init__(
     async def review_pull_request(self, pr_id: int) -> None:
         pull_request = await self._github.get_pull_request(pr_id)
         review = self._analyzer.analyze_code(pull_request)
-        # await self._github.submit_review(pr_id, review)
\ No newline at end of file
+        await self._github.submit_review(pr_id, review)
```

Comment:
**style**: Consider adding error handling around the submit_review call since it's an async operation that could fail. Wrap with try-except and log errors.